packages/env/lib/commands/run.js

```diff
@@ -8,6 +8,7 @@ const path = require( 'path' );
  * Internal dependencies
  */
 const initConfig = require( '../init-config' );
+const { getHostUser } = require( '../build-docker-compose-config' );
```

Comment:
nit: could move this function to its own file in lib/, since it's not super related to duild docker compose config

Reply:
I thought about it but didn't, so, I did it now 😄